.vsts-ci.yml

```
@@ -46,6 +46,13 @@ stages:
 - stage: build
   jobs:
   - job: Publish_Build_Configuration
+    pool:      
+      ${{ if eq(variables['System.TeamProject'], 'public') }}:
+        name: NetCore1ESPool-Svc-Public
```

main shouldn't use the `Svc` pool:

```suggestion
        name: NetCore1ESPool-Public
```

---

When I port this to 6.0.2xx and 6.0.1xx, should I use -Svc pools there?